src/components/marker-table/ContextMenu.js

```diff
@@ -84,6 +96,21 @@ class MarkersContextMenu extends PureComponent<Props> {
     });
   };
 
+  convertStackToString(stack, thread, implementationFilter) {
```

Comment:
please flow type this method properly (both inputs and outputs). Especially you'll want to return a string, and then you'll see something is missing :)

Reply:
also note you could take both `thread` and `implementationFilter` from `this.props` in this method, instead of passing them.